staging/src/k8s.io/apiserver/pkg/server/genericapiserver_graceful_termination_test.go

```diff
@@ -950,6 +1011,7 @@ func newGenericAPIServer(t *testing.T, fAudit *fakeAudit, keepListening bool) *G
 	config, _ := setUp(t)
 	config.ShutdownDelayDuration = 100 * time.Millisecond
 	config.ShutdownSendRetryAfter = keepListening
+	config.ShutdownWatchTerminationGracePeriod = 2 * time.Second
```

Comment:
nit: `// much smaller than normal 30s grace period for other non-long running requests`

Reply:
we just need to enable watch draining here, so any positive value is fine here, added a comment